addons/purchase/models/purchase.py

```diff
@@ -127,6 +126,85 @@ def _compute_invoice(self):
     company_id = fields.Many2one('res.company', 'Company', required=True, index=True, states=READONLY_STATES, default=lambda self: self.env.company.id)
     currency_rate = fields.Float("Currency Rate", compute='_compute_currency_rate', compute_sudo=True, store=True, readonly=True, help='Ratio between the purchase order currency and the company currency')
 
+    reminder_confirmed = fields.Boolean("RFQ Confirmed", default=False, readonly=True, help="True if the reminder email is confirmed by the vendor.")
+    po_confirmed = fields.Boolean("PO Confirmed", default=False, readonly=True, help="True if PO reception is confirmed by the vendor.")
+
+    receipt_reminder_email = fields.Boolean('Receipt Reminder Email',
+        help='Automatically send a reminder email to your vendors before receipt')
+    reminder_date_before_receipt = fields.Integer('Days Before Receipt',
+        help="Number of days to send reminder email before the promised receipt date")
```

Comment:
these two last fields, shouldn't they be settings on the company?

Reply:
apparently they should be related on the res.partner

Reply:
Yeah, I already changed that.

Reply:
@sle-odoo The change requires access rights to res.partner. By default every user should have, but missing the test_access_rights. I added the access rights in the test.